concordia/static/js/base.js

```diff
@@ -68,6 +68,44 @@ function displayMessage(level, message, uniqueId) {
     $messages.append($newMessage);
 }
 
+function displayHTMLMessage(level, message, uniqueId) {
```

Comment:
This is more duplication than I'd like. The way I would probably do this is to rely on the fact that [`jQuery.prepend()`](http://api.jquery.com/prepend/) accepts both HTML strings and elements and refactor this so we have a `displayHtmlMessage()` function which is what we have now except without the `createTextNode` call and then make `displayMessage()` basically just be `function displayNewMessage(level, message, uniqueId) { return displayHtmlMessage(level, document.createTextNode(message), uniqueId); }` so most calls will safely escape HTML but you can call `displayHtmlMessage()` directly when you know you have safe content.